### PR TITLE
Responsiveness

### PR DESCRIPTION
**Pull request by usorfaitheloho**
##Milestone 4: Desktop version

In this milestone, I have created :
- background images for the site
- desktop.css file for media queries on breakpoint 992px for the desktop
- updated styles.css to match styles in desktop.css

### CODE REVIEW

File: styles/utility.css [hunk 1]
```diff
@@ -414,3 +414,241 @@ textarea {
...
+    margin-top: 152px;
+    display: flex;
+    flex-direction: row;
+    position: relative;
+    justify-content: space-around;
+    background-image: url("../images/illustration-2-contact-form-desktop.png");
+    background-repeat: no-repeat;
+    background-position-y: bottom;
+    background-position-x: 20vw;
+  }
```
Comment (Hamzaoutdoors) [inline styles/utility.css:606]: - [x] Great job so far! But unfortunately, you need to stick to the design as much as possible for better UI (user interface) and this is a good professional practice that every frontend software developer should maintain. Please, make sure to give your background image some paddings on the left side so they can look closely similar to the [original design](https://www.figma.com/file/t3EJUCAEViw3QasuJLPLVT/Microverse-Student-Potfolio-Templates-Main?node-id=1%3A1471) 😸 .
- [ ] As you can see in screenshots below you have some overflow following `x-direction` please try to work on this issue by making it hidden

## **Your project**
real

## **Design**
design

**Review by Hamzaoutdoors — CHANGES_REQUESTED** [review]
Hey @usorfaitheloho, I am **Hamza** your reviewer for this PR 😸 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

image

## To Highlight
✔️ The repo looks professional
✔️ Git/GitHub flow best practices have been followed
✔️ There are no linter errors
✔️ You have a descriptive README

## Required Changes ♻️

_See the inline comments below for more required changes 😃_  

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
**N.A**

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -278,11 +292,11 @@ <h5>Skills</h5>
                     </ul>
                 </div>
             </div>
-
+        </div>
         </div>
     </section>
     <section>
```
Comment (brytebee) [inline index.html:298]: - [ ] **Required:** Great job so far! I observed that the following changes still need to be fixed to get you to the next project. The below numbering is marked on your design image attached:

1.  The text in this section is aligned to the centre at the moment. Let's change it to left align.
2. All items here are all aligned to the left as well. as shown in the image.
3. There is a horizontal line before the social icons.
4. Place the `textarea` in front of the SVG to be similar to the Figma template.
5. Add more padding between the form and the footer social icons.

Your Design | Figma Template
:---: | :---:
image | image

I hope this helps 😄

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi, @usorfaitheloho👋, 

Great work on making the changes 👏 👏 
You've done well implementing some of the requested changes requested by the previous review, but there are still some which aren't addressed yet.


# Suggested changes
_Check the comments under the review._




Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by t-yanick — APPROVED** [review]
Hi @usorfaitheloho, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles/utility.css (hunk 1 above)
Comment (usorfaitheloho) [reply styles/utility.css:606]: Hi @Hamzaoutdoors 

I have worked on the overflow and padding issues just as it is in the design.
Thanks for your review

File: index.html (hunk 2 above)
Comment (usorfaitheloho) [reply index.html:298]: Hello @brytebee 
Thanks for your review 😄

**Comment by usorfaitheloho** [comment]
Thanks you so much @t-yanick 😄

**Pull request merged**